.cirrus.yml

```diff
@@ -2,6 +2,7 @@ container:
   image: node:12
 
 build_task:
+  auto_cancellation: false
```

Comment:
Why dependabot is confused when tasks are aborted?

Reply:
It’s actually Cirrus, it commits and then opens the PR, and without this it gets confused and cancels both runs 🤷‍♂

Reply:
(The timing is very close together so I think cirrus has trouble deciding which to cancel)

Reply:
This is a workaround and not a fix. Please create a separate issue describing what's happening with dependabot PRs so we can think of an actual solution.

Reply:
Okay